addons/binding/org.openhab.binding.neato/OSGI-INF/NeatoHandlerFactory.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
Maybe you have to rebase on the current master first, alternative would be to search replace all '2014-2016' by '2010-2017'.